### PR TITLE
tests: require an email appendage to avoid accidentally deleting acco…

### PR DESCRIPTION
**Pull request by xginn8**
…unts

Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

Resolves: # 
HQ: <url> 
See: <url> 
Depends-on: <url> 
Change-type: major|minor|patch 

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

**Comment by xginn8** [comment]
this is a bit belt+suspenders, and i'm open to other approaches. we had a small incident in staging where i accidentally deleted my account, apps, and some shared apps i had access to (ref. https://www.flowdock.com/app/rulemotion/e-cloud-staging/threads/8uu60XY6PGfMeepodSaRta6As89).

**Comment by thgreasi** [comment]
Seems that someone will also have to go to appveyor and update those env vars....

**Comment by thgreasi** [comment]
Let me do that...

File: tests/integration/setup.ts [hunk 1]
```diff
@@ -68,6 +68,19 @@ const buildCredentials = function () {
...
 		},
 	};
+	if (
+		![creds.email, creds.paid.email, creds.register.email].every((email) => {
+			if (email) {
+				return email.includes('+testsdk');
+			} else {
+				return true;
+			}
+		})
```
Comment (thgreasi) [inline tests/integration/setup.ts:78]:
```suggestion
		![creds.email, creds.paid.email, creds.register.email].every((email) => email == null || email.includes('+testsdk'))
```

Also, can we leave the paid user tests out? (I would need to reach to the customer success for this, which would take time, and we don't do any write operations for that user.)

Comment (xginn8) [reply tests/integration/setup.ts:78]: how about i leave it out for now, but we start the process of updating that as well? even though we do no writing now, that may change in the future and unless we enforce it as an invariant i worry about future data loss.

Comment (thgreasi) [reply tests/integration/setup.ts:78]: Sure

Comment (xginn8) [reply tests/integration/setup.ts]: thank you for all your help :heart:

**Review by thgreasi — APPROVED** [review]
:+1: just squash all commits

**Pull request merged**